Network/Packet.h

```diff
@@ -232,7 +232,7 @@ namespace OP2Internal
 		};
 
 		// Member functions
-		int Checksum() const;
+		int Checksum();

```

Comment:
Could you elaborate on the problem here?

The `const` might affect the name mangling, which will affect linking. The decorated symbol name may need to be updated in the .asm file. The expected decorated name will be given in an error message during linking.

OP2Internal.asm: line 492:
```asm
Export 0x00490F10, ?Checksum@Packet@OP2Internal@@QAEHXZ
```

I double checked the function at the listed address `0x00490F10` in Outpost2.exe. It is indeed `const`. It makes not attempt to modify the `this` object. We should try to maintain `const` here, which likely requires updating the decorated symbol name.